core/ledger/kvledger/txmgmt/statedb/statecouchdb/statecouchdb_test.go

```diff
@@ -817,10 +820,10 @@ func TestPaginatedQuery(t *testing.T) {
 		t.Fatalf("Couchdb state impl is expected to implement interface `statedb.IndexCapable`")
 	}
 
-	fileEntries, errExtract := ccprovider.ExtractFileEntries(dbArtifactsTarBytes, "couchdb")
+	dbIndexArtifacts, errExtract := extractFileEntries(dbArtifactsTarBytes)
```

Comment:
Now since this package expect simply a map, there is no need to create the tar file and then untar (and hence no need to copy the function `extractFileEntries` from ccprovider below). You can simply use the map at line 802.

Reply:
Valid point. Done.